utils/litemap/Cargo.toml

```diff
@@ -23,6 +23,7 @@ independent = true
 all-features = true
 
 [dependencies]
+databake = { workspace = true, default-features = false, optional = true }
```

Comment:
nit (nb): would love to switch to explicit features and `dep` syntax in this file while we're at it

Reply:
+1